server/data/scanner.js

```diff
@@ -0,0 +1,153 @@
+const axios = require("axios");
+const nexmo = require("./nexmoConfig");
+
+
+scanner = () => {
+
+    const YOUR_VIRTUAL_NUMBER = "18572560178";
+    const from = "18572560178";
+    let userNumber = "";
+    const URL = "http://localhost:5000/api/sms"; // ********** CHANGE FOR DEPLOYMENT *************
```

Comment:
Does this need to be reverted back?

Reply:
This is part of willies PR when I created mine I created my branch from his. So that I had his changes.

Reply:
This is part of willies PR when I created mine I created my branch from his. So that I had his changes.